resources/js/features/root/pages/AdminFacilitiesList.page.tsx

```diff
@@ -0,0 +1,66 @@
+import { A } from "@solidjs/router";
+import {AUTO_SIZE_COLUMN_DEFS, createTableTranslations} from "components/ui";
+import {TQueryTable} from "components/ui/Table/TQueryTable";
+import {AccessBarrier, useLangFunc} from "components/utils";
+import {Admin} from "data-access/memo-api/groups/Admin";
+import {Component} from "solid-js";
+
+export default (() => {
+  const t = useLangFunc();
+  return (
+    <AccessBarrier roles={["globalAdmin"]}>
+      <TQueryTable
+        mode="standalone"
+        staticPrefixQueryKey={Admin.keys.facilityLists()}
+        staticEntityURL="admin/facility"
+        staticTranslations={createTableTranslations("facilities")}
+        intrinsicColumns={["id", "url"]}
+        additionalColumns={["actions"]}
+        columnOptions={{
+          name: {
+            metaParams: {canControlVisibility: false},
+          },
+          createdAt: {
+            columnDef: {
+              sortDescFirst: true,
+            },
+            metaParams: {
+              filtering: {
+                useDateOnlyInputs: true,
+              },
+            },
+          },
+          updatedAt: {
+            columnDef: {
+              sortDescFirst: true,
+            },
+            metaParams: {
+              filtering: {
+                useDateOnlyInputs: true,
+              },
+            },
+          },
+          actions: {
+            columnDef: {
+              cell: (c) => {
+                return <A href={`/${c.row.getValue("url")}`}>{t("models.facility.page")}</A>
+              },
+              ...AUTO_SIZE_COLUMN_DEFS,
+            },
+          },
+        }}
+        initialColumnsOrder={[
+          "id",
+          "name",
+          "createdAt",
+        ]}
+        initialVisibleColumns={[
```

Comment:
Jak widzę tak sensownie sformatowany kod to mam niestety obawy że nie użyłeś prettiera💩, który często produkuje bezsensownie sformatowany kod, ale obecnie go tu niby stosujemy. Więc `npm run formatter` (czy jakoś tak) albo plugin do VS Code sobie zainstaluj. Może w przyszłości przegłosujemy coś innego ale na razie jest jak jest.

Reply:
✔